cliquet/__init__.py

```diff
@@ -53,8 +56,12 @@ def _redirect_to_version_view(request):
 def set_auth(config):
     """Define the authentication and authorization policies.
     """
+    ttl = int(config.registry.settings.get('fxa-oauth.cache_ttl_seconds',
```

Comment:
Having this named "ttl" can cause confusion in case we have other ttls in there. I would name it with the name of the config, or `oauth_cache_ttl`.